doc/docs.md

```diff
@@ -352,7 +352,16 @@ println(s) // "hello world"
 
 In V, a string is a read-only array of bytes. String data is encoded using UTF-8.
 
-Strings are immutable.
+Strings are immutable by default which means their values cannot be changed like how it is in Go and Java.
```

Comment:
`Strings are immutable by default`
No, they are just immutable. There is no need for the `by default`. 

Sometimes they can be changed in unsafe blocks, but that is very hackish and may not always work, and is reserved for special situations (string literals for example may be stored in read only data sections in the executable, and depending on how your OS runs, writing to read only pages may not be allowed). The language just assumes that strings are immutable.